subsystem/src/main/java/org/wildfly/extension/elytron/tls/subsystem/_private/ElytronTLSLogger.java

```diff
@@ -0,0 +1,400 @@
+/*
+ * Copyright 2022 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.extension.elytron.tls.subsystem._private;
+
+import static org.jboss.logging.Logger.Level.INFO;
+import static org.jboss.logging.Logger.Level.WARN;
+
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.NoSuchProviderException;
+
+import org.jboss.as.controller.ExpressionResolver;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.server.deployment.DeploymentUnitProcessingException;
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+import org.jboss.msc.service.Service;
+import org.jboss.msc.service.ServiceController;
+import org.jboss.msc.service.ServiceName;
+import org.jboss.msc.service.StartException;
+import org.wildfly.security.x500.cert.acme.AcmeException;
+
+/**
+ * Log messages for Elytron TLS subsystem
+ *
+ * @author <a href="mmazanek@redhat.com">Martin Mazanek</a>
+ * @author <a href="mailto:carodrig@redhat.com">Cameron Rodriguez</a>
+ */
+@MessageLogger(projectCode = "ELYTLS", length = 4)
+public interface ElytronTLSLogger extends BasicLogger {
```

Comment:
Since this is a subsystem logger, we could name this `ElytronTLSSubsystemMessages`.